src/main/java/org/fundacionjala/core/constans/ConstantsCore.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.core.constans;
+
+/**
+ * [SL] This class offers constant for the core.
+ */
+public final class ConstantsCore {
+    /**
+     * Constructor.
+     */
+    private ConstantsCore() {
+
+    }
+    /** Navigator Constants. **/
+    public static final String BROWSER_CHROME = "Google Chrome";
+    public static final String BROWSER_CHROMIUM = "Chromium";
+    public static final String BROWSER_MOZILLA = "Mozilla Firefox";
+    public static final String BROWSER_EDGE = "Microsoft Edge";
+
+    /** Browser Properties Constants. **/
+    public static final String BROWSER_VERSION = "version";
+    public static final String BROWSER_IMPLICITWAIT = "implicitWaiting";
+    public static final String BROWSER_EXPLICITWAIT = "explicitWaiting";
+    public static final String BROWSER_SLEEPINGTIME = "sleepingTime";
+    public static final String BROWSER_FLAGS = "flags";
```

Comment:
Why are they necesary to save this constants? they are saved as Browser's class attributes, so to access them we have the getters. Please delete these constants.

Reply:
That is correct,  I delete these constants